docs/SplunkOperatorUpgrade.md

```diff
@@ -23,13 +23,19 @@ A Splunk Operator for Kubernetes upgrade might include support for a later versi
 ```
 wget -O splunk-operator-install.yaml https://github.com/splunk/splunk-operator/releases/download/1.1.0/splunk-operator-install.yaml
 ```
+
+2. Download the upgrade script.
 ​
-2. (Optional) Review the file and update it with your specific customizations used during your install. 
+```
+wget -O upgrade-to-1.1.0.sh https://github.com/splunk/splunk-operator/releases/download/1.1.0/upgrade-to-1.1.0.sh
+```
+
+3. (Optional) Review the file and update it with your specific customizations used during your install. 
```

Comment:
What is file referring to here? It would be nice if we can specify that.

Reply:
I retained this line from previous release. I am not sure if there is any change they need to do. unless they want to change the namespace from `splunk-operator` to something else.